frontend/src/routes/habits/create/+page.svelte

```diff
@@ -68,65 +56,20 @@
               error={form?.errors?.some((error) => error.input == "name")}
             />
 
-            <div class="flex flex-row gap-5">
-              <!-- TODO: add min="0" and max="23" -->
-              <AuthInput
-                name={"start_time_hours"}
-                label={"Start time (hours)"}
-                type={"number"}
-                placeholder={"09"}
-                autocomplete={"given-name"}
-                path={"M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z"}
-                value={form?.startTimeHours ?? ""}
-                error={form?.errors?.some(
-                  (error) => error.input == "start_time_hours",
-                )}
-              />
-
-              <!-- TODO: add min="0" and max="59" -->
-              <AuthInput
-                name={"start_time_minutes"}
-                label={"Start time (minutes)"}
-                type={"number"}
-                placeholder={"30"}
-                autocomplete={"given-name"}
-                path={"M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z"}
-                value={form?.startTimeMinutes ?? ""}
-                error={form?.errors?.some(
-                  (error) => error.input == "start_time_minutes",
-                )}
-              />
-            </div>
-
-            <div class="flex flex-row gap-5">
-              <!-- TODO: add min="0" and max="90" -->
-              <AuthInput
-                name={"duration_minutes"}
-                label={"Duration (minutes)"}
-                type={"number"}
-                placeholder={"05"}
-                autocomplete={"given-name"}
-                path={"M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z"}
-                value={form?.durationMinutes ?? ""}
-                error={form?.errors?.some(
-                  (error) => error.input == "duration_minutes",
-                )}
-              />
+            <TimePicker 
+            bind:_date 
+            label="Start Time" 
+            id="start_time" 
+            error={form?.errors?.some((error) => error.input == "start_time")} 
+            placeholder="hours and minutes"
+            data={form?.startTime} />
 
-              <!-- TODO: add min="0" and max="59" -->
-              <AuthInput
-                name={"duration_seconds"}
-                label={"Duration (seconds)"}
-                type={"number"}
-                placeholder={"00"}
-                autocomplete={"given-name"}
-                path={"M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z"}
-                value={form?.durationSeconds ?? ""}
-                error={form?.errors?.some(
-                  (error) => error.input == "duration_seconds",
-                )}
-              />
-            </div>
+            <TimePicker 
```

Comment:
Even though the field is for minutes:seconds it only allows values up to 24 for minutes